### ISSUE TITLE
Редактирование страниц через админку

### ISSUE THREAD

**Issue by cofob** [fire-square/autodonate#113]
# Это можно сделать 2 путями
## MD-подобный формат
Страницы создаются в формате:
```
# Заголовок
{Widget parameter=value}
```

Для этого потребуется свой парсер. Сложный вариант, но более простой для реализации в стандартной админке и простой для пользователя.

## Относительный формат
Создаётся специальная таблица в БД с полем many-to-many со ссылками на объекты настроек виджетов. Более простой для реализации в коде, нужна продвинутая админка, сложно для пользователя.

**Comment by PerchunPak** [comment]
Почему свой? Почему какая нибудь либа не подойдёт. Уверен у `m2r2` используется такая.

**Comment by PerchunPak** [comment]
Да, `m2r2` использует `mistune==0.8.4`. [Вот документация.](https://mistune.readthedocs.io/en/v0.8.4/)

**Comment by cofob** [comment]
> Да, `m2r2` использует `mistune==0.8.4`. [Вот документация.](https://mistune.readthedocs.io/en/v0.8.4/)

Лучше делать как в #115. Будет сразу нативно.
